jetbrains-ultimate/resources/META-INF/IU.xml

```diff
@@ -7,6 +7,4 @@
     <depends optional="true" config-file="ext-go.xml">org.jetbrains.plugins.go</depends>
     <depends optional="true" config-file="ext-ruby.xml">org.jetbrains.plugins.ruby</depends>
     <depends optional="true" config-file="ext-php.xml">com.jetbrains.php</depends>
-    <depends optional="true" config-file="ext-codewithme.xml">com.jetbrains.codeWithMe</depends>
```

Comment:
Why do we need to move these to a separate file?

Reply:
tests try to load them and fail